README.md

```diff
@@ -75,52 +94,92 @@ Please refer to our [CONTRIBUTING](CONTRIBUTING.md) document.
+    (e.g. agreement/gossip/network.go registers three handlers).
+	 - `rpcs` contains the HTTP RPCs used by `algod` processes to query one
+           another.
+  - `agreement` ([README](agreement/README.md)) contains the agreement service,
+    which implements Algorand's Byzantine Agreement protocol.  This protocol
+    allows participating accounts to quickly confirm blocks in a fork-safe
+    manner, provided that sufficient account stake is correctly executing the
+    protocol.
+  - `node` integrates the components above and handles initialization and
+    shutdown.  It provides queries into these components.
 
 `daemon` defines the two daemons which provide Algorand clients with services:
 
-  - `daemon/algod` holds the `algod` daemon, which implements a participating node.  `algod` allows a node to participate in the agreement protocol, submit and confirm transactions, and view the state of the Algorand Ledger.
-    - `daemon/algod/api` ([README](daemon/algod/api/README.md)) is the REST interface used for interactions with algod.
-  - `daemon/kmd` ([README](daemon/kmd/README.md)) holds the `kmd` daemon.  This daemon allows a node to sign transactions.  Because `kmd` is separate from `algod`, `kmd` allows a user to sign transactions on an air-gapped computer.
+  - `daemon/algod` holds the `algod` daemon, which implements a participating
+    node.  `algod` allows a node to participate in the agreement protocol,
+    submit and confirm transactions, and view the state of the Algorand Ledger.
+         - `daemon/algod/api` ([README](daemon/algod/api/README.md)) is the REST
+           interface used for interactions with algod.
+  - `daemon/kmd` ([README](daemon/kmd/README.md)) holds the `kmd` daemon.  This
+    daemon allows a node to sign transactions.  Because `kmd` is separate from
+    `algod`, `kmd` allows a user to sign transactions on an air-gapped computer.
   
 The following packages allow developers to interface with the Algorand system:
 
   - `cmd` holds the primary commands defining entry points into the system.
-    - `cmd/catchupsrv` ([README](cmd/catchupsrv/README.md)) is a tool to assist with processing historic blocks on a new node.
+	 - `cmd/catchupsrv` ([README](cmd/catchupsrv/README.md)) is a tool to
+           assist with processing historic blocks on a new node.
   - `libgoal` exports a Go interface useful for developers of Algorand clients.
   - `debug` holds secondary commands which assist developers during debugging.
   
 The `auction` package implements the Algorand auctions.
 
-The following packages contain tools to help Algorand developers deploy networks of their own:
+The following packages contain tools to help Algorand developers deploy networks
+of their own:
 
   - `nodecontrol`
   - `tools`
   - `docker`
-  - `commandandcontrol` ([README](commandandcontrol/README.md)) is a tool to automate a network of algod instances.
+  - `commandandcontrol` ([README](commandandcontrol/README.md)) is a tool to
```

Comment:
this directory was just moved under test; it would be nice if you'll update that here as well.

Reply:
updated to test/commandandcontrol/README.md